src/libraries/System.Net.Quic/src/System/Net/Quic/Internal/ReceiveBuffers.cs

```diff
@@ -52,7 +52,7 @@ public int CopyFrom(ReadOnlySpan<QUIC_BUFFER> quicBuffers, int totalLength, bool
             int totalCopied = 0;
             for (int i = 0; i < quicBuffers.Length; ++i)
             {
-                Span<byte> quicBuffer = quicBuffers[i].Span;
+                ReadOnlySpan<byte> quicBuffer = quicBuffers[i].Span;
```

Comment:
This is fine, but what does it fix?

Reply:
it's a leftover change from a previous version, I will roll it back